### PR TITLE
Event list style changes #115

### PR DESCRIPTION
**Pull request by githubsrki**
### Closes Issues:
#115
### New packages:
 implementation "com.jcminarro:RoundKornerLayouts:0.4.0" has been added
https://github.com/JcMinarro/RoundKornerLayouts

### CODE REVIEW

File: app/src/main/java/com/bigneon/studio/util/AppUtils.kt [hunk 1]
```diff
@@ -37,15 +37,15 @@ class AppUtils {
         }
 
         fun getCurrentTimestamp(): String {
-            val dateFormat = SimpleDateFormat(DATE_FORMAT, Locale.US)
+            val dateFormat = SimpleDateFormat(DATE_FORMAT_MS, Locale.US)
```
Comment (Krakaw) [inline app/src/main/java/com/bigneon/studio/util/AppUtils.kt]: The timezone is in the venue object in the events list.

**Comment by Krakaw** [comment]
Please also include a screenshot with your PR's

File: app/src/main/java/com/bigneon/studio/util/AppUtils.kt (hunk 1 above)
Comment (githubsrki) [reply app/src/main/java/com/bigneon/studio/util/AppUtils.kt]: https://github.com/big-neon/bn-android-doorperson/pull/119/commits/d92f9cdfd0a3040ff0c7c7d55e7dd2fa34d773fa

**Pull request merged**